sql/src/main/java/com/vmware/ddlog/ir/DDlogEApply.java

```diff
@@ -34,18 +34,31 @@
 public class DDlogEApply extends DDlogExpression {
     private final String func;
     private final List<DDlogExpression> args;
+    // If true the suffix notation with dot is preferred.
+    private boolean suffix;
```

Comment:
Is this used in this PR anywhere? If not, what's an intended use-case?

Reply:
Not yet used, but may be, the result is easier to read.